packages/docs/package.json

```diff
@@ -6,6 +6,7 @@
   "repository": "CMSgov/design-system",
   "dependencies": {
     "@cmsgov/design-system-core": "^3.6.1",
+    "@cmsgov/design-system-layout": "^3.6.1",
```

Comment:
Is this something temporary that will hopefully not be needed after we have new build scripts? Why is this needed?

Reply:
The docs package still needs to import the old packages because the new package isnt published yet, so we should be able to remove it after we publish `@cmsgov/design-system`. I added it back because the docs were broken in the branch to start with haha